index.php

```diff
@@ -37,8 +37,8 @@
 */
 
 // Check PHP version not to have trouble
-if (version_compare(PHP_VERSION, "5.4.0") < 0) {
-   die("PHP >= 5.4.0 required");
+if (version_compare(PHP_VERSION, GLPI_MIN_PHP) < 0) {
+   die(sprintf("PHP >= %s required", GLPI_MIN_PHP));
 }
```

Comment:
This won't work as GLPI_MIN_PHP not yet defined.
(as this check is done before everything else, to avoid blank page)